pkgs/applications/video/mpv/scripts/occivink.nix

```diff
@@ -10,24 +11,30 @@ let
   in stringAsChars (c: if isUpper c != null then "-${toLower c}" else c);
 
   mkScript = name: args:
-    buildLua (lib.attrsets.recursiveUpdate rec {
+    let self = rec {
       pname = camelToKebab name;
+      version = "unstable-2022-10-02";
       src = fetchFromGitHub {
         owner = "occivink";
         repo = "mpv-scripts";
         rev = "af360f332897dda907644480f785336bc93facf1";
         hash = "sha256-KdCrUkJpbxxqmyUHksVVc8KdMn8ivJeUA2eerFZfEE8=";
       };
-      version = "unstable-2022-10-02";
+      passthru.updateScript = unstableGitUpdater {};
 
       scriptPath = "scripts/${pname}.lua";
 
       meta = with lib; {
         homepage = "https://github.com/occivink/mpv-scripts";
         license = licenses.unlicense;
         maintainers = with maintainers; [ nicoo ];
+
+        # Sad hack needed to make `common-updaters` work here
+        position = with builtins;
+          let pos = unsafeGetAttrPos "version" self;
+          in "${pos.file}:${toString pos.line}";
```

Comment:
It is possible to set `pos` passed to `mkDerivation` instead?

https://github.com/NixOS/nixpkgs/blob/bd464981e07c97f47585366a47f7cd35e7f7726c/pkgs/stdenv/generic/make-derivation.nix#L174-L179